modules/op-uninstalling-the-pipelines-operator.adoc

```diff
@@ -15,3 +15,8 @@ You can uninstall the {pipelines-title} Operator by using the *Administrator* pe
 . Click the *{pipelines-title}* Operator tile. The Operator tile indicates that the Operator is installed.
 
 . In the *{pipelines-title}* Operator description page, click *Uninstall*.
+
+[WARNING]
+====
+When you uninstall the {pipeline-shortname} Operator, all resources within the target namespace where the Operator is installed, the `oepenshift-pipelines` namespace by default, are lost including the secrets you configured with it.
```

Comment:
typo in namespace name

Reply:
Seen it.
Addressed.

Reply:
Can merge request be made now @mramendi ?